tasks/compile-example.js

```diff
@@ -297,6 +297,7 @@ module.exports = function(config, indexPath, updateTimestamp) {
       if (document.isAmpStory) {
         generateStoryPreviewEmbed(stream, example, args, {
```

Comment:
The `generateStoryPreviewEmbed` function seems to have only 2 arguments? Am I reading that right?

Reply:
fixed